datastore/builtin/build_test.go

```diff
@@ -0,0 +1,148 @@
+package builtin
+
+import (
+	"github.com/drone/drone/common"
+	"github.com/franela/goblin"
+	"testing"
+)
+
+func TestBuildstore(t *testing.T) {
+	db := mustConnectTest()
+	bs := NewBuildstore(db)
+	cs := NewCommitstore(db)
+	defer db.Close()
+
+	g := goblin.Goblin(t)
+	g.Describe("Buildstore", func() {
+
+		// before each test we purge the package table data from the database.
+		g.BeforeEach(func() {
+			db.Exec("DELETE FROM builds")
+			db.Exec("DELETE FROM commits")
+		})
+
+		g.It("NewBuildstore()", func() {
+			//Add build
+			build := common.Build{
+				ID:       1,
+				CommitID: 1,
+				State:    "success",
+			}
+			err1 := bs.SetBuild(&build)
+			g.Assert(err1 == nil).IsTrue()
+		})
+
+		g.It("Build()", func() {
+			build_list := []*common.Build{
```

Comment:
the names used in `g.It` should read more like sentences. The reason is that when tests fail it will print a nice, human readable sentence to the console describing what broke.

you can see an example of what this looks like here:
https://raw.githubusercontent.com/franela/goblin/master/goblin_output.png

see the user tests for reference:
https://github.com/drone/drone/blob/0.4.0/datastore/builtin/users_test.go#L31